.editorconfig

```diff
@@ -0,0 +1,18 @@
+# EditorConfig http://EditorConfig.org
+
+# Project Root
+root = true
+
+# Default Code Style
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 4
+indent_style = tab
```

Comment:
yes, I'd allow an exception for Makefiles, but I really don't want tabs as the default